kubernetes/assets/dashboards/kubernetes_deployments.json

```diff
@@ -0,0 +1,1345 @@
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Replicas Updated",
+            "type": "timeseries",
+            "width": 59,
+            "x": 60,
+            "y": 57
+        },
+        {
+            "height": 12,
+            "legend": false,
+            "legend_size": "0",
+            "tile_def": {
+                "custom_links": [],
+                "requests": [
+                    {
+                        "q": "sum:kubernetes_state.deployment.rollingupdate.max_unavailable{$scope,$kube_cluster_name,$kube_namespace,$kube_deployment} by {deployment}",
+                        "style": {
+                            "palette": "warm",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "area"
+                    }
+                ],
+                "right_yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                },
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Replicas Unavailable During Rolling Update",
+            "type": "timeseries",
+            "width": 59,
+            "x": 60,
+            "y": 72
+        },
+        {
+            "height": 12,
+            "tile_def": {
+                "autoscale": true,
+                "precision": 0,
+                "requests": [
+                    {
+                        "aggregator": "last",
+                        "conditional_formats": [
+                            {
+                                "comparator": ">",
+                                "palette": "red_on_white",
+                                "value": "0"
+                            },
+                            {
+                                "comparator": "<=",
+                                "palette": "green_on_white",
+                                "value": "0"
+                            }
+                        ],
+                        "q": "sum:kubernetes_state.deployment.rollingupdate.max_unavailable{$scope,$kube_deployment,$kube_cluster_name,$kube_namespace}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "time": {},
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Rolling Update",
```

Comment:
I removed this widget completely (since I misunderstood this metric before). I don't think it makes sense as an aggregated in the overview. Instead I added a Query Value widget for the number of Deployments